llama_index/retrievers/fusion_retriever.py

```diff
@@ -125,6 +126,22 @@ def _run_async_queries(
 
         return results
 
+    async def _run_async_queries_async(
```

Comment:
nit: naming is confusing. We can just refactor this to `_arun_queries`? 

and maybe adjust the naming of `_run_async_queries` and `_run_sync_queries` as appropriate as well.

Reply:
Agreed